package.json

```diff
@@ -78,6 +83,7 @@
         "pluralize": "8.0.0",
         "prettier": "2.1.2",
         "prettier-plugin-java": "0.8.2",
```

Comment:
@mishima Can you update prettier-plugin-java to 0.8.3?

Reply:
Done